ogre2/src/Ogre2RenderEngine.cc

```diff
@@ -612,6 +616,10 @@ void Ogre2RenderEngine::CreateResources()
 
   Ogre::ArchiveManager &archiveManager = Ogre::ArchiveManager::getSingleton();
 
+  Ogre::Archive *customizationsArchiveLibrary =
+      archiveManager.load( rootHlmsFolder + "Hlms/Ignition", "FileSystem",
```

Comment:
for the path please use `common::JoinPath()`

Reply:
cc04f5e7729648934f855f881a4d6cd128f7b39c